glean-core/android/src/test/java/mozilla/telemetry/glean/pings/CustomPingTest.kt

```diff
@@ -190,17 +193,15 @@ class CustomPingTest {
             uploadEnabled = true,
         )
 
-        // There should be no ping upload worker,
-        // because there is no ping to upload.
-        assertFalse(getWorkerStatus(context, PingUploadWorker.PING_WORKER_TAG).isEnqueued)
-
-        // But if the custom ping is specifically submitted,
+        // If the custom ping is specifically submitted,
         // it should be received.
         customPing.submit()
         // Trigger work manager once.
         // This should launch one worker that handles all pending pings.
         triggerWorkManager(context)
         var request = server.takeRequest(2L, TimeUnit.SECONDS)!!
+        // Assert we only got the one:
+        assertNull(server.takeRequest(2L, TimeUnit.SECONDS))
```

Comment:
Note that this means a 2s wait time here now (slower tests), but I guess it's our only option.

Reply:
Yup. Unless someone wants to build a way for the Kotlin LB to synchronously determine whether pings are submitted